src/main/java/core/basesyntax/service/FileReader.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileReader {
+    List<String> dataFromFile(String filePath);
```

Comment:
method names must be verbs
```suggestion
    List<String> readFromFile(String filePath);
```